pkg/kademlia/kademlia.go

```diff
@@ -888,6 +888,34 @@ func (k *Kad) ClosestPeer(addr swarm.Address, skipPeers ...swarm.Address) (swarm
 	return closest, nil
 }
 
+// IsWithinDepth returns if an address is within the neighborhood depth of a node
```

Comment:
Use `.` at the end of the comment to represent the whole sentence. https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

This goes for other comments in this PR.